letsencrypt-auto-source/pieces/fetch.py

```diff
@@ -56,7 +57,9 @@ def get(self, url):
 
         """
         try:
-            return self._opener.open(url).read()
+            // socket module docs say default timeout is None: that is, no
```

Comment:
*gasps*

Erik's been cheating on Python! That doesn't look like Pythonic comment syntax to me.

Reply:
How embarrassing! I thought it didn't look right, but I couldn't put my finger on it!